### DISCUSSION TITLE
AzuraCast for WebTV / Videostream Scheduling

### DISCUSSION THREAD

**Discussion by schwabing** [AzuraCast/AzuraCast#4596] [General]
Hi Community,
could AzuraCast be extended to run a WebTV service with scheduled videostreams, generated from a Video on demand library? Think of pluto.tv

- Let's say you have Vimeo Pro, or brightcove, or kaltura to manage your videos
- as a Video DJ you want to assemble playlists from the catalog of the above video platforms and schedule them as shows
- the output is streamed using liqudsoap or alike as a pseudo livestream

In the end it's exactly what AzuraCast does for audio, but with a video API as a source instead of mp3 uploads or paths, and hls videostreaming as output instead of audio. Too far fetched or could this be done with a plugin?

**Comment by schwabing** [comment]
I hope you don't mind me bumping this topic. Azurast does exactly for music what I'd need for video: create multiple stations with "linear", broadcast-style playback of mp4 (video on demand) content. And live livestreams, but this is for later.
So let me phase my most important question differently: do you see a way to work with an external digital asset management system (DAM, DAM API) instead of a files on a local drive?

**Comment by Mozitux** [comment]
I recently became interested in this possibility of making a webtv with Azuracast, but that would require far too heavy modifications for it to work properly.

I recommend that you instead look at OBS Studio's playlist plugins as well as RTMP-HSL technologies. Everything can work on a Linux server provided you have a GPU or very good processors and excellent bandwidth.

**Comment by clintguy74** [comment]
If you have a server, I have a setup using owncast and OBS, its in the federation and is not restricted. use that for webTV, but Azuracast is the Audio Goto for Radio station..
